quest_test/test_resource_stream.py

```diff
@@ -1,11 +1,19 @@
+import uuid
+
 import pytest
+from pathlib import Path
 
-from src.quest import step, Historian
+from src.quest import step, create_filesystem_manager
```

Comment:
For tests we can use the historian directly. Or at least use an in-memory manager.